tests/spec/core/biblio-spec.js

```diff
@@ -145,4 +146,17 @@ describe("W3C — Bibliographic References", () => {
     expect(first).toMatch("[a]");
     expect(last).toMatch("[Zzz]");
   });
+
```

Comment:
I wonder if we need a test also that clears the biblio database?

Reply:
not sure what and how would we test using that :confused:

Reply:
so, you could just add the check to the "normalizes aliases" test above... but just clear the db, and create a new ReSpec doc.